spec/index.bs

```diff
@@ -7450,6 +7524,9 @@ All depth formats support {{GPUTextureUsage/COPY_SRC}}, {{GPUTextureUsage/COPY_D
 
 None of the depth formats can be filtered.
 
+The {{GPUTextureAspect/"depth-only"} aspect of all of the depth formats can be sampled
```

Comment:
```suggestion
The {{GPUTextureAspect/"depth-only"}} aspect of all of the depth formats can be sampled
```

Reply:
Why on earth doesn't `bikeshed` complain about this?